src/components/SendNativeToken/sendNativeTokenUI.tsx

```diff
@@ -1,25 +1,58 @@
+/* MIT License
+ * 
+ * Copyright (c) [Year] [Author]
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
 import React, { useState, KeyboardEvent, useEffect } from 'react';
+
+// Services
 import {
   EtherspotBatches,
   EtherspotBatch,
   EtherspotTransaction,
   useEtherspotTransactions,
 } from "@etherspot/transaction-kit";
-import { ethers } from "ethers";
+
+// Utils
 import { isValidEthereumAddress } from '../../utils/validation';
 
+// Props interface for the SendNativeTokenUI component
+/**
+ * @typedef {Object} SendNativeTokenUIProps - Props for the SendNativeTokenUI component.
+ * @property {string} receiverAddress - The Ethereum address of the receiver.
+ * @property {number} chain - The Ethereum chain ID.
+ * @property {string} [className] - Additional CSS class for styling.
+ * @property {React.CSSProperties} [style] - Inline styles for the component.
+ * @property {boolean} [unstyled=false] - Flag indicating whether to apply default styles.
+ * @property {boolean} [debug=false] - Flag indicating whether to enable debugging.
+ * @property {React.ReactNode | React.RefObject<HTMLElement>} [triggerElement] - Trigger element for UI interactions.
+ * @property {boolean} [disableSendOnEnter=false] - Flag indicating whether to disable sending on Enter key press.
+ * @property {boolean} [onlyEstimate=false] - Flag indicating whether to only estimate the transaction.
+ */
 
-interface SendNativeTokenProps {
-  receiverAddress: string;
-  chain: number;
-  className?: string;
-  style?: React.CSSProperties;
-  unstyled?: boolean;
-  debug?: boolean;
-  triggerElement?: React.ReactNode | React.RefObject<HTMLElement>;
-  disableSendOnEnter?: boolean;
-  onlyEstimate?: boolean;
-}
+/**
+ * SendNativeTokenUI component for sending native tokens using the EtherspotTransactionKit.
```

Comment:
This needs to be more detailed. Explain this as if you're explaining to someone who knows nothing about sending a native crypto token to another address. This is who this library is aimed at.